packages/server/src/scim.ts

```diff
@@ -96,6 +106,41 @@ export class ScimServer implements DirectoryProvider {
         await this._startScimServer();
     }
 
+    private async _getScimOrg(orgId: OrgID) {
+        const prov = await this.storage.get(OrgProvisioning, orgId);
+        return (prov.metaData?.scim || { users: [], groups: [] }) as ScimOrg;
+    }
+
+    private async _saveScimOrg(orgId: OrgID, scimOrg: ScimOrg) {
+        const prov = await this.storage.get(OrgProvisioning, orgId);
+        prov.metaData = prov.metaData || {};
+        prov.metaData.scim = scimOrg;
```

Comment:
I figured the `metaData` property on `OrgProvisioning` was a good place as any to store this. We **could** also store this in a separate table though.